order/README.md

```diff
@@ -9,7 +9,7 @@ Ordering with loosely defined order values will give the provider more freedom t
 ## Request
 | Field Name | Type                                                                       | Description |
 | ---------- | -------------------------------------------------------------------------- | ----------- |
-| datetime       | string                                                                     | **REQUIRED.** Datetime field is a [ISO8601 Time Interval](https://en.wikipedia.org/wiki/ISO_8601#Time_intervals) |
+| datetime       | string                                                                     | **REQUIRED.** Single date+time, or a range ('/' separator), formatted to [RFC 3339, section 5.6.](https://tools.ietf.org/html/rfc3339#section-5.6) Use double dots .. for open date ranges. \[Copied from [STAC API spec](https://github.com/radiantearth/stac-api-spec/tree/release/v1.0.0/item-search#query-parameter-table)\] |
```

Comment:
```suggestion
| datetime       | string                                                                     | **REQUIRED.** Single date and time, or a range ('/' separator), formatted to [RFC 3339, section 5.6.](https://tools.ietf.org/html/rfc3339#section-5.6) Use double dots `..` for open date ranges. Only one boundary can use double dots. |
```

Reply:
What does providing a single datetime mean?

Reply:
Good question. I imagine it means the same as providing `..` as the second part. But maybe we need to define a new data type here that mandates both parts. *Or*... as previously suggested - make this an array that requires length of 2. This seems like a stronger argument now

Reply:
For reference: https://docs.ogc.org/is/17-069r3/17-069r3.html#_parameter_datetime @captaincoordinates

Reply:
@m-mohr PR updated and includes some wording from the OGC Features API spec related to dates (which supports `..` or empty strings for open date ranges).